### ISSUE TITLE
Type 'VideoFrame' is not assignable to type 'Transferable'

### ISSUE THREAD

**Issue by davidstrahm** [microsoft/TypeScript#53432]
# Bug Report
Typescript does not let me Use VideoFrame as a transferable object when passing it to a webworker.

See #36823, its the same for 'OffscreenCanvas' (fixed)

### 🔎 Search Terms
VideoFrame, Transferable Objects Webcodecs

### 🕗 Version & Regression Information
Typescript Version 4.9.5

### 💻 Code
```typescript
const video= document.createElement("video")
const videoFrame = new VideoFrame(video);
const worker = new Worker(URL.createObjectURL(new Blob([``],{type: "text/javascript"})))
worker.postMessage(videoFrame,[videoFrame])
```

### 🙁 Actual behavior
An error is thrown by the compiler:
`Type 'VideoFrame' is not assignable to type 'Transferable'`

### 🙂 Expected behavior
https://developer.mozilla.org/en-US/docs/Web/API/VideoFrame

> VideoFrame is a [transferable object](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Transferable_objects).

It should not throw an error.

**Comment by RyanCavanaugh** [comment]
I'm not seeing an error in nightly. Seems like this has probably been fixed.
